examples/computer_administrators_script.py

```diff
@@ -0,0 +1,21 @@
+from subprocess import Popen
+from pywinauto import application
+import time
+
+Popen(['compmgmt.msc'], shell = True)
+
+app = application.Application().connect(path="mmc.exe")

```

Comment:
It would be more reliable if it waited for the process start. I can suggest reuse our brand new ```always_wait_until_passes``` decorator
```python
@always_wait_until_passes(4, 2, application.ProcessNotFoundError)
def connect_to_mmc():
    return application.Application().connect(path="mmc.exe")
app = connect_to_mmc()
```